R/modify_pd.R

```diff
@@ -0,0 +1,43 @@
+#' Update indention information of parse data
+#'
+#' @param pd A nested or flat parse table that is already enhanced with
+#'   line break and space information via [create_filler] or
+#'   [create_filler_nested].
+#' @param indent_by How many spaces should be added after the token of interest.
+#' @name update_indention
+NULL
+
+#' @rdname update_indention
+indent_round <- function(pd, indent_by) {
+  start <- which(pd$token == "'('") + 1
```

Comment:
Well, do you mean `stop != nrow(pd) - 1`? 
Probably not in the case of round brackets. However, I wanted to keep the function rather general, since later on, we probably want to use a closure that returns functions like `indent_round`, but also one that indents after operators such as `%>%`.
`data_frame(x = 1, y = 2) %>%` 
__`  do_one_thing()` 

Here, we want `stop = nrow(pd)`, which is different from the round brackets, where probably always `stop != nrow(pd) -1 `. 
But anyways, I think you are right, in the end we might just use two different closures then, i.e. one for backets and one for other operators such as `%>%` `+` etc.